### ISSUE TITLE
Total balances do not reflect amounts in transactions list

### ISSUE THREAD

**Issue by nop33** [alephium/alephium-frontend#981]
@h0ngcha0 reported to me today that he and a friend of his experienced the problem that even though the desktop wallet displays a list of incoming confirmed transactions, the total balance of the wallet in the overview page remains `0`.

I've seen a similar issue on Telegram where a user complained that even though their transaction is displayed in the transaction list as well as on our explorer, the balance of their wallet has not updated, even after waiting for 45 minutes. @diomark has experienced this as well.

**Comment by nop33** [comment]
The issue was the `skipToken` in the ALPH balance query. By simplifying it in the above commit the issue is resolved.

To reproduce it, we sent some ALPH to an address of the wallet that is not the default one.

**Comment by nop33** [comment]
Reopening this issue because it still affects some users. Particularly, these have been the new reports:
1. Case 1: Transferred NFT from an external address to the wallet. Wallet still displays _"The wallet doesn't have any NFTs. NFTs of all your addresses will appear here."_
2. Case 2: Exchanged meme token for ALPH. Transaction appeared in wallet, but ALPH amount didn't update.

**Comment by nop33** [comment]
I discovered the reason why people are reporting that their desktop wallet balances don’t update after transactions are confirmed. It’s because the backend does not respond fast enough to these changes.
1. When a tx is sent, we poll the `GET /transactions/{txHash}` endpoint
2. When the endpoint returns (meaning the tx is confirmed), we immediately call the following endpoints:
   - `GET /addresses/{addressHash}/balance`
   - `GET /addresses/{addressHash}/tokens-balance`

I noticed that these endpoints return results that are not up to date.
